scipy/io/tests/test_netcdf.py

```diff
@@ -111,10 +111,14 @@ def test_read_write_files():
                 check_simple(f)
 
         # Read file from fileobj, with mmap
-        with open('simple.nc', 'rb') as fobj:
-            with netcdf_file(fobj, mmap=True) as f:
-                assert_(f.use_mmap)
-                check_simple(f)
+        with suppress_warnings() as sup:
+            if IS_PYPY:
+                sup.filter(RuntimeWarning,
+                           "Cannot close a netcdf_file opened with mmap=True.*")
+            with open('simple.nc', 'rb') as fobj:
+                with netcdf_file(fobj, mmap=True) as f:
+                    assert_(f.use_mmap)
```

Comment:
Could maybe switch to plain `assert` now that we are using pytest, since pytest replaces plain assert with a bunch of useful stuff for error handling & prevents the removal of plain asserts by the python optimization flag. Conversely, not sure that we want a campaign of converting all the `assert_` to `assert` anytime soon.

Reply:
+1 for switching to `assert ` statements in new code